src/components/Create/components/pages/NftRewards/NftRewardsPage.tsx

```diff
@@ -94,6 +119,36 @@ export const NftRewardsPage = () => {
             <CreateCollapse>
```

Comment:
Not a huge issue, but would be great if opening this section didn't cause the whole section to expand slightly:

https://user-images.githubusercontent.com/96150256/214214050-6bf9a26a-f630-4005-ac4e-a67312e8a6fc.mp4

Reply:
There's something causing the divs to not fully size to 600px it seems. Will open an issue to look into, but on first look I can't seem to find it

Reply:
https://github.com/jbx-protocol/juice-interface/issues/2885